test/specs/new/setext_no_blankline.html

```diff
@@ -0,0 +1,12 @@
+<pre><code>indented code block
+</code></pre>
+<p>=</p>
+<p>fenced code block</p>
+<pre><code>=
+</code></pre>
+<blockquote><h1>blockquote</h1>
```

Comment:
The `<h1>` here doesn't match what I'm seeing in Commonmark. Should be `<p>` instead.

https://spec.commonmark.org/dingus/

Reply:
hmm ya it looks like common mark treats blockquote differently based on if it has a `>` at the beginning of the line.

[demo 1](https://spec.commonmark.org/dingus/?text=%3E%20blockquote%0A%3D)
[demo 2](https://spec.commonmark.org/dingus/?text=%3E%20blockquote%0A%3E%20%3D)

The spec says the block quote will continue without lines starting with `>` only if the content is [paragraph continuation text](https://spec.commonmark.org/0.31.2/#paragraph-continuation-text). And it gives [Example 234](https://spec.commonmark.org/0.31.2/#example-234) as an example of not continuing the block quote.

In this case the `=` would not be paragraph continuation text since it indicates a setext heading without the content above being a blockquote.

Reply:
Looks like https://github.com/commonmark/commonmark-spec/issues/539 seems to say it is paragraph continuation text simply because common mark says so.

This is one of those rules where the common mark spec states how commonmark works, not how markdown should work. 😕

This is how marked works [currently](https://marked.js.org/demo/?outputType=html&text=%3E%20blockquote%0A%3D&options=%7B%0A%20%22async%22%3A%20false%2C%0A%20%22breaks%22%3A%20false%2C%0A%20%22extensions%22%3A%20null%2C%0A%20%22gfm%22%3A%20true%2C%0A%20%22hooks%22%3A%20null%2C%0A%20%22pedantic%22%3A%20false%2C%0A%20%22silent%22%3A%20false%2C%0A%20%22tokenizer%22%3A%20null%2C%0A%20%22walkTokens%22%3A%20null%0A%7D&version=12.0.0) anyway so it isn't a regression.

I will create an L2 bug for this but I don't think it should block this PR.